deploy-authdevs.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-envvalue=("authdev1" "authdev2")
+envvalue=("authdev1" "authdev2" "new authdev1" "new authdev2")
```

Comment:
can we please have this without spaces. something like `authdev{1,2}-ng` or similar

Reply:
only because i don't like the idea of weird dangling spaces in 'parameters'